pkg/compiler/analysis.go

```diff
@@ -14,6 +14,7 @@ var (
 	builtinFuncs = []string{
 		"len", "append", "SHA256",
 		"SHA1", "Hash256", "Hash160",
+		"VerifyECDSA",
```

Comment:
Do we really want to name it this way? It not that it's a bad name, but I think it would be better for us to closely follow C# naming scheme where it's just a `VerifySignature`.